lib/inspec/resources/service.rb

```diff
@@ -141,7 +141,7 @@ def select_service_mgmt # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticCom
         elsif version > 0
           SysV.new(inspec, service_ctl || "/usr/sbin/service")
         end
-      when "redhat", "fedora", "centos", "oracle", "cloudlinux"
+      when "redhat", "fedora", "centos", "oracle", "cloudlinux", "scientific"
```

Comment:
I'm not at a laptop to check, but I think the `os` hash will contain the "families" the OS is part of, and a more generic solution might be to make this switch block work by family and not specific OSs.

To avoid complicating things, it'd also be fine to just add `scientific` for now

Reply:
Does inspec have a concept for platform_family like chef? We bring in scientific and oracle for free in most of our logic by just saying 'rhel' which includes redhat and all the direct clones.

Reply:
I believe we support both the `os[:family]` and `platform_family` options but will need to double confirm next time I'm pried in